r/R/dplyr-mutate.R

```diff
@@ -57,7 +64,7 @@ mutate.arrow_dplyr_query <- function(.data,
     if (inherits(results[[new_var]], "try-error")) {
       msg <- handle_arrow_not_supported(
         results[[new_var]],
-        format_expr(exprs[[i]])
+        format_expr(exprs2[[i]])
```

Comment:
we return the original expression in case of error, `"lubridate::as_datetime()"`, instead of `"lubridate_as_datime()"` to avoid confusion.